static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -1491,6 +1499,13 @@ const StyledField = styled(Field)<{extraMargin?: boolean}>`
   }
 
   margin-bottom: ${p => `${p.extraMargin ? '60px' : space(1)}`};
+  display: inline;
```

Comment:
once you move the margin you should be able to use the `inline` Field prop instead

Reply:
I've kept the space since its needed for small screen size, for some reason I can't get css children selectors to work out